data/scripts/creaturescripts/player/update_client_on_advance.lua

```diff
@@ -0,0 +1,17 @@
+local updateClientOnAdvance = CreatureEvent("Update Client On Advance")
```

Comment:
I would remove this file completely

Reply:
Also the new file name and titles have less semantic

Reply:
the script does exactly what the name says and now works as it should, there is no point in changing it.

Reply:
Not at all. And please do not resolve my conversation if there is an open discussion.

Reply:
That script is not doing "exactly waht the name says"

Reply:
If the script wasdoing "exactly what is says" it should be "update_client_on_advance_level_plus_take_screenshot_on_advance_any_skill"

Reply:
IMO, still not enough reason to merge those files in one

Reply:
Oh great you hid my reply

Reply:
If you're not satisfied with my approach, make a request yourself and let the moderators decide.
You are being very annoying to someone who never or almost never contributes anything.

Reply:
https://github.com/otland/forgottenserver/pulls?page=1&q=is%3Apr+author%3Aomarcopires ??